evaluation/gaia/run_infer.py

```diff
@@ -140,11 +148,22 @@ def process_instance(instance, agent_class, metadata, reset_logger: bool = True)
     if state is None:
         raise ValueError('State should not be None.')
 
-    model_answer = ''
+    model_answer_raw = ''
     for act, _ in reversed(state.history):
-        if isinstance(act, MessageAction) and act.source == 'agent':
-            model_answer = act.content
+        if isinstance(act, CmdRunAction) and act.source == 'agent':
+            model_answer_raw = act.thought
```

Comment:
Should we also add break here?

Reply:
good catch!! fixed in the new commit - will now do an auto-merge